pyproject.toml

```diff
@@ -138,3 +138,37 @@ score = "no"
 [tool.hdev]
 project_name = "lms"
 project_type = "application"
+
+[tool.mypy]
+python_version = 3.11
+
+disable_error_code = [
```

Comment:
These the errors that we have in the codebase now, nothing special about them.

If we merge a version of this the goal would be to remove them one by one.

Reply:
You'll have to add this to the cookiecutter or to `.cookiecutter/includes/pyproject.toml`